lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/failed.yaml

```diff
@@ -16,6 +16,14 @@ states:
           ticfCriBeta:
             targetState: CRI_TICF_BEFORE_NO_MATCH
 
+  FAILED_COI:
+    events:
+      next:
+        targetState: COULD_NOT-CONFIRM-DETAILS
+        checkFeatureFlag:
+          ticfCriBeta:
+            targetState: CRI_TICF_BEFORE_NO_MATCH
```

Comment:
I think this might need (annoyingly) a new state, something like `CRI_TICF_BEFORE_COULD_NOT_CONFIRM_DETAILS`, which calls TICF and then forwards the user to the `COULD_NOT_CONFIRM_DETAILS` state.
TICF is annoying...